Makefile

```diff
@@ -0,0 +1,17 @@
+all: build
+
+install_deps:
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(shell go env GOPATH)/bin v1.45.2
+
+lint:
+	golangci-lint run --print-issued-lines=false --fix ./...
```

Comment:
Does this also fix issues? Then better rename the task to `lint-fix` and create a separate task just for linting